pkg/test/nodeclass.go

```diff
@@ -69,6 +81,106 @@ func ApplyDefaultStatus(nodeClass *v1alpha2.AKSNodeClass, env *coretest.Environm
 	nodeClass.SetConditions(conditions)
 }
 
+func ApplyCIGImages(nodeClass *v1alpha2.AKSNodeClass) {
+	ApplyCIGImagesWithVersion(nodeClass, DefaultCIGImageVersion)
+}
+
+func ApplyCIGImagesWithVersion(nodeClass *v1alpha2.AKSNodeClass, cigImageVersion string) {
+	nodeClass.Status.Images = []v1alpha2.NodeImage{
+		{
+			ID: fmt.Sprintf("/CommunityGalleries/AKSUbuntu-38d80f77-467a-481f-a8d4-09b6d4220bd2/images/2204gen2containerd/versions/%s", cigImageVersion),
+			Requirements: []corev1.NodeSelectorRequirement{
+				{
+					Key:      "kubernetes.io/arch",
```

Comment:
Should be able to use existing constants for label keys (here and elsewhere)

Reply:
Updated usages here and others to use consts. Good callout